src/golang/cmd/server/handler/v2/node_artifact_result_content_get.go

```diff
@@ -106,9 +100,9 @@ func (h *NodeArtifactResultContentGetHandler) Perform(ctx context.Context, inter
 	args := interfaceArgs.(*nodeResultGetArgs)
 	emptyResp := &nodeResultGetResponse{}
 
-	dag, err := h.DAGRepo.GetByDAGResult(
+	dag, err := h.DAGRepo.Get(
 		ctx,
-		args.nodeResultID,
+		args.dagID,
```

Comment:
Great! Can you also make the change for the other `*_content_get` endpoints as well?